docs/proposals/Release-Supply-Chain-Metadata.md

```diff
@@ -0,0 +1,178 @@
+# Supply Chain Metadata for Ratify Assets
+
+Ratify currently publishes multiple forms of release assets both for production and development uses. Currently, these assets are not published with accompanying supply chain metadata including signatures, SBOMs, and provenance. Shipping each of these forms of metadata with all binaries and container images produced by Ratify will provide consumers a verifiable way to guarantee integrity of Ratify assets. Furthermore, this will improve Ratify's OSSF scorecard.
+
+## What does Ratify currently publish?
+
+Ratify publishes two types: release and development. Release assets accompany official Ratify Github releases. Development assets are published weekly (or adhoc as needed).
+
+Each publish type includes the following group of assets:
+
+* CRD container image to `ghcr.io/ratify-project`
+* Base container image to `ghcr.io/ratify-project`
+* Base + plugins container image to `ghcr.io/ratify-project`
+* Ratify binaries as a single bundle per OS/arch which includes:
+  * ratify binary
+  * sbom-verifier plugin binary
+  * vulernability-report verifier plugin binary
+* `checksums.txt` which contains list of checksum for each binary bundle
+* Helm chart
+  * Release ONLY: published packaged helm chart to `ratify-project.github.io`
+  * Dev ONLY: published packaged helm chart to `ghcr.io/ratify-project`
+* Source code (zip + tar ball)
+
+## Signatures
+
+Ratify should support signing all container images and binaries with both Sigstore Cosign and Notary Project.
+
+### Notary Project Image Signatures
+
+Ratify can utilize Notary Project's install and sign github actions published.
+Notation requires a KMS to store the signing certificate. There is no support for Github Encrypted Secrets for storing the certificate.
+
+1. Generate a self-signed certificate stored in Azure Key Vault
+2. Configure Notation action to sign using the cert stored in AKV
+3. Sign CRD, base, plugin images
+4. Store the public verification certificate at `ratify.dev/.well-known/ratify-verification.crt` & `github.com/ratify-project/ratify/dev/.well-known/ratify-verification.crt`
```

Comment:
Should we consider `ratify.dev/.well-known/pki-validation/ratify-verification.crt`?

Reply:
See also https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml

Reply:
Sounds good. I'll add the `pki-validation` sub path